FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Mutual_Auth/demo_config.h

```diff
@@ -180,17 +180,44 @@
  * #define democonfigCLIENT_PASSWORD    "...insert here..."
  */
 
+/**
+ * @brief The name of the operating system that the application is running on.
+ * The current value is given as an example. Please update for your specific
+ * operating system.
+ */
+#define democonfigOS_NAME                   "Windows"
+
+/**
+ * @brief The version of the operating system that the application is running
+ * on. The current value is given as an example. Please update for your specific
+ * operating system version.
+ */
+#define democonfigOS_VERSION                "10"
+
+/**
+ * @brief The name of the hardware platform the application is running on. The
+ * current value is given as an example. Please update for your specific
+ * hardware platform.
+ */
+#define democonfigHARDWARE_PLATFORM_NAME    "PC"
```

Comment:
We shouldn't necessarily hard-code "PC" for the customer
```suggestion
#define democonfigHARDWARE_PLATFORM_NAME    "PC"
```

Reply:
What would we hardcode it to? This demo only runs on WinSim.

Reply:
Good point